.storybook/manager.js

```diff
@@ -1,6 +1,12 @@
 import { addons } from "@storybook/addons"
 import theme from "./theme"
+import favicon from "../src/assets/eth-home-icon.png"
```

Comment:
```suggestion
import favicon from "../src/assets/favicon.png"
```

Agree with discussion, I think we might as well just use the favicon asset in this case. Other than that looks good to go